packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -55,34 +62,24 @@ $skeleton-color-end: rgba($gray1, 0.2) !default;
   }
 }
 
-// Some properties should not be overriden. A more specific explanation is given before each usage
-// of "!important".
+// This class hides content with a glowing, rounded rectangle. CSS properties that should not be overriden by consumers use the
```

Comment:
i suppose i was thinking of this the other way, as in consumer styles go "on top" of BP styles. i can reword!